cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <router-link
-    :to="{ name: 'credentials', params: { vaultId } }"
     v-if="type === 'regular'"
+    :to="{ name: 'credentials', params: { vaultId } }"
```

Comment:
Was this shuffle was added by prettier ??

Reply:
Linter